node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -88,3 +90,87 @@ func TestTokenListEmptyCoinGeckoId(t *testing.T) {
 		assert.Greater(t, len(tokenConfigEntry.coinGeckoId), 0)
 	}
 }
+
+// If not true, then there exists some chain that Wormhole governs yet has no governed tokens. That means assets
+// native to the chain are not actually governed and can be transferred across all other supported chains without
+// increasing their Governor limit. We don't want this.
+func TestAllGovernedChainsHaveGovernedAssets(t *testing.T) {
+
+	// Build a list of chains that are not governed we will filter these out. Compare with `mainnet_chains.go`
+	// When a new ChainID is added, it must either be added to this list or else have entries populated in
+	// one of the list of tokens accessed by `tokenList()`.
+	ignoredChains := map[vaa.ChainID]bool{
+		// Pyth is special
+		vaa.ChainIDPythNet: true,
+		// BTC is special
+		vaa.ChainIDBtc: true,
+		// Wormchain is a Wormhole abstraction over IBC type networks
+		vaa.ChainIDWormchain: true,
+		// From the perspective of the Governor, all of these are "Wormchain" because they use IBC.
+		vaa.ChainIDCosmoshub:  true,
+		vaa.ChainIDEvmos:      true,
+		vaa.ChainIDKujira:     true,
+		vaa.ChainIDNeutron:    true,
+		vaa.ChainIDCelestia:   true,
+		vaa.ChainIDStargaze:   true,
+		vaa.ChainIDSeda:       true,
+		vaa.ChainIDDymension:  true,
+		vaa.ChainIDProvenance: true,
+		// ID reserved but not yet Governed
+		vaa.ChainIDBerachain: true,
+		vaa.ChainIDSnaxchain: true,
+		// Testnets
```

Comment:
This seems like a maintenance headache. When a new chain goes to testnet, I would need to add it here. Later, when the chain goes to mainnet, someone needs to remember to come back here and take it out.

Reply:
Yep! Evan raised the same point and I agree this isn't a great implementation. I'm going to read from https://github.com/wormhole-foundation/wormhole/blob/main/node/pkg/governor/mainnet_chains.go#L11 chainList instead and see if that feels better. That way we only care about things that 1) have to do with the Governor and 2) are on Governor's mainnet.